### ISSUE TITLE
Track history outside of combat

### ISSUE THREAD

**Issue by amir-arad** [manuelVo/foundryvtt-drag-ruler#263]
I would love the ability to track movement history also outside of combat mode. 

As in my game (OSE) I already have an "End turn" macro that handles all sorts of things outside of combat (time passes, resources spent etc.), If I could just add a line there that resets all movement history each time a turn passes, that would be a complete solution for me.

**Comment by manuelVo** [comment]
In your use case, how would Drag Ruler decide when to track the movement history and when not to?

**Comment by amir-arad** [comment]
no, In OSE combat the time keeping is different: a turn is 10 minutes but a combat round is 10 seconds

**Comment by amir-arad** [comment]
a tyrn is 10 minutes, combat round is 10 seconds.
there are 60 rounds in a turn.

there is also a different speed for rncounter and exploration, and a combat is always an encounter.

I would like to keep track on turns progression as the characters move.
